app/controllers/posts_controller.rb

```diff
@@ -1,5 +1,5 @@
 class PostsController < ApplicationController
-  def show
-    @post = Post.new(current_user.token, params[:title], params[:post_id])
+  def index
+    @post = Post.build(current_user.token, params[:subreddit_name], params[:post_id])
```

Comment:
Curious as to why you moved from `.new` to `.build`. I don't have a strong opinion on it, so I'm just intrigued.